### ISSUE TITLE
Emphasis on Webb Assets

### ISSUE THREAD

**Issue by seeddisperser** [webb-tools/webb-experiences#55]
# Overview
To simplify the existing design sprints, and to emphasis the value add of this product, I propose two ideas:
1. shift emphasis onto interacting with Webb assets
2. combine bridge and account page (this will make sense shortly)

# Background
The assumption that has guided the design process has been that bridging is made more valuable by zero-knowledge cross chains capabilities. However, this gets it backwards: it is that zero-knowledge cross-chain pools which enable a number of valuable actions, including bridging. The existing designs prioritize bridging patterns. Yet the value adds such as multiple chains, unwrapping into different assets, etc, all come from Webb assets. 

**So what would happen if the emphasis was given to the shielded Webb assets.**

# Emphasis on Webb Assets

Here's a view of how a person could be presented with Webb assets they can deposit into using the available assets in their wallet. This contextualizes the deposit/transfer/withdrawal process, while also surfacing key info such as underlying assets, supported chains, liquidity, etc.

![Image](https://user-images.githubusercontent.com/109543061/192901586-a66eedc7-d3cf-4c56-a2c8-10b91914e4ea.png)

# Combine Bridge and Account 

I realized that if users interact with deposit/withdrawal/transfer controls by navigating a list of webb assets, the account page could be presented in the exact same way:

![Image](https://user-images.githubusercontent.com/109543061/192902366-75172fc4-bb2d-4d7e-a5bb-2df360a38345.png)

# Putting the pieces together

### Try out a [quick prototype](https://www.figma.com/proto/nr7RB0ENJYji4Dc8P6klfi/Asset-Representation-(micro-sprint)?page-id=22%3A1085&node-id=41%3A7477&viewport=-4522%2C2117%2C0.25&scaling=scale-down&starting-point-node-id=41%3A7477) of this idea. 

This is what I'm imagining a full screen would look like 

![Image](https://user-images.githubusercontent.com/109543061/192902791-3cb1293d-62e1-4816-a830-8d35bc056815.png)

The user would also be able to click into a detail view of Webb assets for additional information such as note history, data viz, number of transactions over a period of time, ect. 

![Image](https://user-images.githubusercontent.com/109543061/192902955-51f92878-2fde-4138-95f4-f716dbf44be5.png)

# tldr
To simplify the existing design sprints, and to emphasis the value add of this product, I propose two ideas:
1. shift emphasis onto interacting with Webb assets. This contextualizes the deposit/transfer/withdrawal process, while also surfacing key info such as underlying assets, supported chains, liquidity, etc.
2. combine bridge and account, making "my assets" a tab on the bridge page

**Comment by monaiuu** [comment]
I like this line of thinking; I think it gives more clarity and contexts for users. Would love to hear feedback from the team.

**Comment by nepoche** [comment]
I really like the idea of a focus on the shielded pools instead of a focus on the bridging action. I agree that the UX seems more cohesive with the combination of note account and bridge pages.

I think the table component on the "my shielded assets" alongside the card components for shielded assets is a great way to inform the user of available actions for the currently connected chain.

Looking at the mockups:
- I could see presenting 'additional information' could get a bit cluttered. Maybe we can consider separate stats page for shielded assets (similar to info.uniswap.org?)
- The only indication of "Bridge" functionality would be the chain-grouped avatars in a shielded asset's component.  Could there be additional ways we can inform about the bridge?

**Comment by seeddisperser** [comment]
I think these are both good points to consider as we work on the interface. To rephrase as how might we questions,
HMW:
- indicate "Bridge" functionality clearly and obviously?
- present the information a user needs to make transactions without getting cluttered?

**Comment by monaiuu** [comment]
On the note "HMW indicate "Bridge" functionality clearly and obviously?" — I think it shouldn't be an issue if we continue with the side navigation component which indicates to users they are on the bridge UI. And the education card on top is also a good place to inform users of the bridge functionality. 
<img width="1066" alt="Screen Shot 2022-09-30 at 1 03 39 PM" src="https://user-images.githubusercontent.com/109539270/193348085-f39b76ef-0c93-46a6-835e-fdc48a33efbf.png">

**Comment by monaiuu** [comment]
Looking at the uniswap stats page for individual LPs, curious to know what sort of data would be different or need to be distinguished in webb's shielded pools compared to normal LPs? @nepoche
